python/setup.py

```diff
@@ -251,7 +251,7 @@ def get_packages(self):
             "grpcio >= 1.32.0; python_version < '3.10'",  # noqa:E501
             "grpcio >= 1.42.0; python_version >= '3.10'",  # noqa:E501
             "opencensus",
-            "pydantic < 2",  # 2.0.0 brings breaking changes
+            "pydantic >= 2.5.0",
```

Comment:
Would it make sense to do `pydantic >= 2.5, < 2" here so we can support both? (given that the code seems to have already been written with that in mind and we seem to have tests for both)

That's also _roughly_ what fastAPI does: https://github.com/tiangolo/fastapi/blob/master/pyproject.toml#L44

Reply:
That makes sense, thanks for the catch! I updated the version.